### PR TITLE
[rosdep] adding python-netaddr for recent ubuntu

### PR DESCRIPTION
**Pull request by asmodehn**

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
Can you please also add artful and bionic? Thanks!

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1657,6 +1657,11 @@ python-netaddr:
     trusty_python3: [python3-netaddr]
     utopic: [python-netaddr]
     vivid: [python-netaddr]
+    xenial: [python-netaddr]
+    yakkety: [python-netaddr]
+    zesty: [python-netaddr]
+    artful: [python-netaddr]
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: sorry I should have specified: the entries have to be in alphabetical order

Comment (asmodehn) [reply rosdep/python.yaml]: ha I forgot as well, sorry I ll fix that.

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks for iterating

**Review by tfoote — APPROVED** [review]

**Pull request merged**